content/guides/installing-python/_index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Installing Python
+---
+
+Python is the external MobiFlight scripting language, e.g., needed to connect custom aircraft APIs to the WinWing CDUs.
+
+**Required Python environment (all three are checked by MobiFlight):**
+
+- Minimum installed Python version: `v3.10`
+- Python path is set in the Windows PATH system variable.
+- Installed Python packages: `websockets, gql, SimConnect`
+
+## Installation instructions
+
+{{% steps %}}
+
+### [Download](https://www.python.org/downloads/) and install Python
+
+> [!WARNING]
+> **Do not forget to set checkmark for adding python.exe to PATH**
+
+{{< screenshot image="python-installer.png" title="Python installer." >}}
+
+### Install the following Python packages via command line
+
+(Regardless of your current directory, simply run the command in your terminal.)
+
+```text
+pip install websockets gql simconnect
+```
+
+List and check all installed packages and versions with:
+
+```text
+pip freeze
+```
+
+The minimum required version for the websockets package is v14.0. If the output from `pip freeze` shows a lower version, update with:
+
+```text
+pip install websockets --upgrade
+```
```

Comment:
Can this simplify down to just one command:

```
pip install websockets gql simconnect --upgrade
```

The average user has no idea how to use pip, and won't know how to interpret the freeze output. 

Also, I suggest using `powershell` instead of `text` as the code language, as that's what users are likely to get and makes for a nice pretty presentation on the page.